test/unit/region.spec.js

```diff
@@ -1042,6 +1050,11 @@ describe('region', function() {
     it('should return the region', function() {
       expect(this.itemView.getRegion('MyRegion')).to.equal(this.region);
     });
+
+    it('should call render if getRegion is called without being rendered', function() {
+      this.itemView.getRegion('whoCares');
```

Comment:
I use an arbitrary region because it shouldn't matter whether the region exists or not.  It will render the view regardless.